effects.h

```diff
@@ -47,17 +49,16 @@ using Game_object_weak = std::weak_ptr<Game_object>;
 class Effects_manager {
 	Game_window *gwin;      // Handy pointer.
 	Special_effect *effects;    // Sprite effects, projectiles, etc.
-	Text_effect *texts;     // Text snippets.
+	std::list<std::unique_ptr<Text_effect>> texts;     // Text snippets.
 public:
-	Effects_manager(Game_window *g) : gwin(g), effects(nullptr), texts(nullptr)
+	Effects_manager(Game_window *g) : gwin(g), effects(nullptr), texts{}
```

Comment:
I missed this before, but it is better to remove the trivial initializer for `texts` altogether, as the default constructor for list is called anyway.

Reply:
That might my MISRA bias ;)
Anyway, fixed.